scripts/bytecodecompare/storebytecode.sh

```diff
@@ -82,6 +82,12 @@ EOF
 
     if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]
     then
+        if !(test -f report.txt)
```

Comment:
Not sure, would be nice to have a distinct message for this and a separate message if the report already existed.